examples/rt633/Cargo.toml

```diff
@@ -1,3 +1,5 @@
+[workspace]
```

Comment:
Why is this a workspace? It's the only crate in this directory. Same Q for the other example crates.

Reply:
This and the other crates in `example` otherwise think they're part of the parent workspace, which causes issues as they define binaries with the same names (e.g. both this and `std` define a `keyboard` binary). This forces them to be in their own separate workspace, which isolates their respective `target` directories and avoids the conflict. I do suspect there's a better way to restructure this so they can be part of the parent workspace, but that's a more invasive change.